html/components/SimpleHtml.js

```diff
@@ -25,6 +25,24 @@ import getEmptyObjectKeys from '../services/getEmptyObjectKeys';
 import isValidVideoFormat from '../services/isValidVideoFormat';
 import Image from './Image';
 
+const UNSPLASH_IMAGE_CONTAINER_ID = 'unsplash-image-container';
+
+function alterUnsplashImagesNode(node) {
+  const styleAttrib = _.get(node, 'attribs.style', '').trim();
+
+  if (node.name === 'div' && node.attribs.id === UNSPLASH_IMAGE_CONTAINER_ID) {
+    const imageContainerWithMargins = {
+      attribs: {
+        style: `${styleAttrib}margin-top: 15px; margin-bottom: 30px;`,
```

Comment:
What happens if `styleAttrib` already contains `margin-top` or `margin-bottom`? Is this a possible situation?

Reply:
I'll check it out.
I'm 99,9% sure it'd work, because it'd read last margin-top & margin-bottom properties, but still... I'll check.